cms/envs/common.py

```diff
@@ -444,7 +444,6 @@
 ENTERPRISE_CONSENT_API_URL = LMS_INTERNAL_ROOT_URL + '/consent/api/v1/'
 FRONTEND_LOGIN_URL = LOGIN_URL
 FRONTEND_LOGOUT_URL = lambda settings: settings.LMS_ROOT_URL + '/logout'
-derived('FRONTEND_LOGOUT_URL')
```

Comment:
Why did this need to be removed?  The settings value was a lambda to accommodate this line

Reply:
Good point, I think that was a poor work around. I'm adding `FRONTEND_LOGOUT_URL` back to `lms/envs/test.py`. It was added there with a comment that it should be removed as a part of this work, but it looks like we need it for cms unittests to pass.